TYPO3.Flow/Tests/Unit/Security/Cryptography/RsaWalletServicePhpTest.php

```diff
@@ -42,8 +42,9 @@ public function setUp()
     {
         vfsStream::setup('Foo');
         $settings['security']['cryptography']['RSAWalletServicePHP']['keystorePath'] = 'vfs://Foo/EncryptionKey';
+        $settings['security']['cryptography']['RSAWalletServicePHP']['paddingAlgorithm'] = OPENSSL_PKCS1_OAEP_PADDING;
         $settings['security']['cryptography']['RSAWalletServicePHP']['openSSLConfiguration']['digest_alg'] = 'sha1';
-        $settings['security']['cryptography']['RSAWalletServicePHP']['openSSLConfiguration']['private_key_bits'] = 384;
+        $settings['security']['cryptography']['RSAWalletServicePHP']['openSSLConfiguration']['private_key_bits'] = 1024;
```

Comment:
Just being curious: Is that change directly linked to the security vulnerability or a general improvement?

Reply:
With the changed padding the key size used for the tests was too small to fit the encrypted data. The size in "real life" is larger by default, at least nowadays.

Reply:
Makes sense, thanks!